driver.go

```diff
@@ -592,3 +592,53 @@ func (d *CCEDriver) setNodeCount(ctx context.Context, clusterInfo *types.Cluster
 	logrus.Info("set cluster node count success")
 	return nil
 }
+
+func (d *CCEDriver) RemoveLegacyServiceAccount(ctx context.Context, clusterInfo *types.ClusterInfo) error {
+	clientSet, err := getClientSet(ctx, clusterInfo)
+	if err != nil {
+		return err
+	}
+
+	return util.DeleteLegacyServiceAccountAndRoleBinding(clientSet)
+}
+
+func (d *CCEDriver) ETCDSave(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented")
+}
+
+func (d *CCEDriver) ETCDRestore(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented")
+}
+
+func getClientSet(ctx context.Context, clusterInfo *types.ClusterInfo) (*kubernetes.Clientset, error) {
```

Comment:
This logic is duplicated in `PostCheck`, please refactor out the common logic (I'd probably have `PostCheck` call this func).